lerna.json

```diff
@@ -0,0 +1,6 @@
+{
+  "lerna": "3.10.6",
+  "version": "independent",
```

Comment:
Currently packages are a bit disconnected (local cli at 1.1 and global at 2.0) so until we reach parity Lerna will ask you which version would you like to bump to